src/MegaPoker.sol

```diff
@@ -19,17 +19,19 @@
 pragma solidity ^0.6.12;
 
 contract PokingAddresses {
-    // OSMs and Spotter addresses
-    address constant eth            = 0x81FE72B5A8d1A857d176C3E7d5Bd2679A9B85763;
+    // OSMs
     address constant btc            = 0xf185d0682d50819263941e5f4EacC763CC5C6C42;
-    address constant univ2daiusdc   = 0x25D03C2C928ADE19ff9f4FFECc07d991d0df054B;
+    address constant eth            = 0x81FE72B5A8d1A857d176C3E7d5Bd2679A9B85763;
+    address constant reth           = 0xeE7F0b350aA119b3d05DC733a4621a81972f7D47;
     address constant wsteth         = 0xFe7a2aC0B945f12089aEEB6eCebf4F384D9f043F;
+
     address constant crvv1ethsteth  = 0xEa508F82728927454bd3ce853171b0e2705880D4;
     address constant guniv3daiusdc1 = 0x7F6d78CC0040c87943a0e0c140De3F77a273bd58;
     address constant guniv3daiusdc2 = 0xcCBa43231aC6eceBd1278B90c3a44711a00F4e93;
+    address constant univ2daiusdc   = 0x25D03C2C928ADE19ff9f4FFECc07d991d0df054B;
+
+    // Spotter
     address constant spotter        = 0x65C79fcB50Ca1594B025960e539eD7A9a6D434A3;
-    address constant reth           = 0xeE7F0b350aA119b3d05DC733a4621a81972f7D47;
```

Comment:
note, sorted these lexigraphically so less likely to miss. e.g. `reth` has moved above